lib/extended-page-mod.js

```diff
@@ -436,23 +442,29 @@ const ExtendedPageModManager = Registry.resolve({
       return;
     }
 
-    for (let rule in RULES)
-      if (RULES[rule].test(document.URL)) {
-        // If the content script is already injected we want to just exit
-        if (window.__contentScriptInjected) {
-          return;
-        }
+    if (window.__contentScriptInjected) return;
```

Comment:
nit.  don't one-line if statements that involve transferring code control. it's easier to miss them